mikado-app/src/viewmodel/displayLayerStore.js

```diff
@@ -125,6 +125,11 @@ class DisplayLayerOperations {
    */
   #backwardConnections = {};
 
+  /**
+   * ID of the node that is currently impending connection, or that was most recently impending connection.
+   */
+  #impendingConnectionNodeID = null;
```

Comment:
Where is this variable being used?

Reply:
Oops, ended up not needing it but forgot to remove.